tests/e2e/yamcs/realtimeData.e2e.spec.mjs

```diff
@@ -249,6 +251,101 @@ test.describe('Realtime telemetry displays', () => {
             }
         });
 
+        test.only('Open MCT does not drop telemetry when a burst of telemetry arrives', async ({ page }) => {
+            const PARAMETER_VALUES_COUNT = 60;
+            /**
+             * A failure mode of the previous implementation of batching was when bursts of telemetry from a parameter arrived all at once.
+             * A burst of 60 messages will overwhelm a per-parameter telemetry buffer of 50, but will not overwhelm a larger shared buffer.
+             */
+
+            // Disable real playback. We are going to inject our own batch of messages
+            await disableLink(yamcsURL);
+
+            const callNumber = await page.evaluate(async () => {
```

Comment:
Can you update this variable name and doc?

Reply:
Yep, both could be better here, will do.